### ISSUE TITLE
Column Heights

### ISSUE THREAD

**Issue by mragon2** [jacobjma/PyQSTEM#14]
Hi, I read in your paper that the model is able to output the heights of the atomic columns, but when I try to load an image and run it through the CNN, the only output is the confidence map.  The code for the calculation of the column heights has already been implemented? If so, how can I see the heights ?
Thanks.

**Comment by jacobjma** [comment]
Hi, are you talking about this repo? https://gitlab.com/schiotz/NeuralNetwork_HRTEM

The column height algorithm is not currently implemented in a public repository. We have not evaluated the algorithm experimentally, but we are continuing to develop the method.

**Comment by jacobjma** [comment]
Yes, the released implementation of the algorithm can only identify peaks. The reliability of the column height identification is still uncertain and needs more verification. 

If you are committed to trying to make it work, I could share some of the experimental code. However, you are in for a potentially very big project.

I am currently just focusing on the peak finding, as this has been proven on experimental data.

**Comment by mragon2** [comment]
Dear Jacob,
thank you very much for your reply. I have a last question about the model.
Are the images generated during training and visible in the folder 'debug'
representative of the  ground truth obtained through the Gaussian
distribution?
I tried to debug the file 'labels.py', to understand how the labels are
defined and I found a good similarity between the numerical values of the
labels and the values on the color bar of the generated pictures.
 I have a matrix of zero labels for the large part of the pixels, and
values around 1 for the pixels where the atomic column should be located,
same as the picture generated when running the file 'ktrain.py'.
Moreover, I guess that the pictures show the same column height for each
cluster since a small rotation is applied in the generation of the images
in the file 'make_cluster_training_data_100.py'?. I tried to impose a large
rotation and I have clusters with different values in the color bar. I
guess this is because a rotation results in seeing columns with different
number of atoms from the perspective of the xy plane.

Thanks,
Marco
